lb.tf

```diff
@@ -67,7 +67,7 @@ resource "aws_lb_target_group" "lb_targets" {
   port                 = 80
   protocol             = "HTTP"
   vpc_id               = var.vpc_id
-  target_type          = "instance"
+  target_type          = var.lb_target_type
```

Comment:
Once again, could automate this part if it has to be a specific value for fargate.

Reply:
Updated